x-pack/legacy/plugins/apm/public/components/shared/charts/TransactionCharts/index.tsx

```diff
@@ -48,12 +49,26 @@ const ShiftedEuiText = styled(EuiText)`
 `;
 
 export class TransactionCharts extends Component<TransactionChartProps> {
-  public getResponseTimeTickFormatter = (t: number) => {
-    return asMillis(t);
+  public getMaxY = (responseTimeSeries: TimeSeries[]) => {
+    const coordinates = flatten(
+      responseTimeSeries.map((serie: TimeSeries) => serie.data as Coordinate[])
+    );
+
+    const numbers: number[] = coordinates.map((c: Coordinate) =>
+      c.y ? c.y : 0
+    );
+
+    return Math.max(...numbers, 0);
+  };
+
+  public getResponseTimeTickFormatter = (formatter: TimeFormatter) => {
```

Comment:
This needs to be imported. It's currently not exported in https://github.com/elastic/kibana/blob/ad111bed1f7cd1c9484f82c9008b77594ba49997/x-pack/legacy/plugins/apm/public/utils/formatters.ts#L111-L113

However, I suggest you rename `TimeFormatter` to `TimeFormatterBuilder` and instead export a new version of `TimeFormatter`:

```
export type TimeFormatter = (
  value: FormatterValue,
  options?: FormatterOptions
) => string;
type TimeFormatterBuilder = (max: number) => TimeFormatter;
```

Reply:
Aha, I missed that. Thanks! Pr is updated.